.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Any notes on the 2 empty lines before the first heading?

Reply:
Some men just like to see the world 🔥

Reply:
..or you know... oversight and such and such.

Reply:
Removed it 😉

Reply:
Sorry for my OCD. 💚